src/Cache/Cache.php

```diff
@@ -108,22 +108,45 @@ class Cache
 
     /**
      * Returns the Cache Registry instance used for creating and using cache adapters.
+     *
+     * @return \Cake\Core\ObjectRegistry
+     */
+    public static function getRegistry()
+    {
+        if (!static::$_registry) {
+            static::$_registry = new CacheRegistry();
+        }
+
+        return static::$_registry;
+    }
+
+    /**
+     * Returns the Cache Registry instance used for creating and using cache adapters.
+     * Also allows for injecting of a new registry instance.
+     *
+     * @param \Cake\Core\ObjectRegistry|null $registry Injectable registry object.
+     * @return void
+     */
+    public static function setRegistry(ObjectRegistry $registry)
+    {
+        static::$_registry = $registry;
+    }
+
+    /**
+     * Returns the Cache Registry instance used for creating and using cache adapters.
      * Also allows for injecting of a new registry instance.
      *
+     * @deprecated Use getRegistry() and setRegistry() instead.
```

Comment:
Yes, I'm not 100% sure but I haven't seen any version in these lines before. Were they added recently?